api/apiFunctions.ts

```diff
@@ -70,7 +70,10 @@ export async function getAllWeeklyPicks(): Promise<
       appwriteConfig.databaseId,
       '66313025000612a5380e',
     );
-    return response;
+
+    // parse and return the weekly user picks
```

Comment:
m: remove. Not needed. Function is self explanatory.

Reply:
✅ Done!